app/apps/cases/views.py

```diff
@@ -50,9 +54,113 @@ class CaseViewSet(
     RetrieveUpdateDestroyAPIView,
     FinesMixin,
     DebriefingsMixin,
-    CaseEventsMixin,
```

Comment:
Volgens mij is de CaseEventsMixin nog wel nodig voor de cases/{id}/case-events endpoint.

Reply:
ziet er uit dat er iets misgegaan is met een merge aan mijn kant. Ik pas het aan! (was event -> caseevent in mijn lokaal branch gemerged)